### PR TITLE
Documentation + UI for location permission

### PR DESCRIPTION
**Pull request by cgntnr**
Title says it all, I added some documentation and also changed the look of the location permission activity to make it coherent with the rest of the app.

### CODE REVIEW

File: app/src/main/res/layout/activity_location_permission.xml [hunk 1]
```diff
@@ -8,23 +8,58 @@
...
+        android:textAlignment="center"
+        android:textColor="#000000"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.533"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
```
Comment (Lollipoke) [inline app/src/main/res/layout/activity_location_permission.xml:25]: Maybe add a contentDescription for the image ?

**Review by Lollipoke — APPROVED** [review]
Great work ! Maybe adding a contentDescription for the image could be good, but other than that I didn't see anything worth mentioning :)

File: app/src/main/res/layout/activity_location_permission.xml (hunk 1 above)
Comment (cgntnr) [reply app/src/main/res/layout/activity_location_permission.xml:25]: You're right. Added it now.

**Review by cmsigrist — APPROVED** [review]
Looks all good to me!

**Pull request merged**